### DISCUSSION TITLE
Can someone help with building COLMAP from source?

### DISCUSSION THREAD

**Discussion by ViniBreda** [colmap/colmap#1259] [Q&A]
Hey guys, I'm fairly new to computer vision and I've been trying to build COLMAP from source to use it as a Lib in C++. Could someone guide me? I've been getting stuck on every pre requisit for a long time and it's taking a long time. I just need to get it working so I can import it to either Embarcadero or MSVS.

**Comment by EstebanDugueperoux2** [comment]
Hi @ViniBreda ,

Could you give more details on your build context?
Personnaly I can build dev branch on Fedora Linux with GCC13 by installing before egein, flann, freeimage and glew.
If you are on Windows, you can try using vcpkg.

Regards.
